doc/cvodes/guide/source/Usage/SIM.rst

```diff
@@ -4138,6 +4138,22 @@ integration tolerances for quadrature variables.
      * ``CV_ILL_INPUT`` -- One of the input tolerances was negative.
 
 
+.. c:function:: int CVodeQuadSVtolerances(void * cvode_mem, realtype reltolQ, N_Vector abstolQ)
+
+   The function ``CVodeQuadSVtolerances`` specifies scalar relative and absolute  tolerances.
+
+   **Arguments:**
+     * ``cvode_mem`` -- pointer to the CVODES memory block.
+     * ``reltolQ`` --  tolerances is the scalar relative error tolerance.
```

Comment:
```suggestion
     * ``reltolQ`` --  is the scalar relative error tolerance.
```
and similarly above for `CvodeQuadSStolerances`. IDAS has the same typo in both of these functions as well.